plugin/fixmyjs.vim

```diff
@@ -28,6 +28,21 @@ if !exists('g:fixmyjs_engine')
     let g:fixmyjs_engine = 'eslint'
 endif
 
+let s:project_root_path = substitute(system("git rev-parse --show-toplevel"), '\n\+$', '', '')
+func! s:get_rc_paths(filename)
+  return map([s:project_root_path . "/" . a:filename, '$HOME/'.a:filename, '$HOME/.vim/'.a:filename], 'expand(v:val)')
+endfun
+
+func! s:some_exist(paths)
+  for s:path in a:paths
+    if filereadable(s:path)
+      return 1
```

Comment:
Can we just return `s:path` here? Then `Line #197` is not needed, basically they are doing the same thing. And at `Line #61`, we could assign the returned `s:path` to an variable and use it at `Line #199`.

And also, we could normalize the value of `g:fixmyjs_rc_filename`, so that it's always an array when we use it. For example: `Line#49-#54`, we could give`g:fixmyjs_rc_filename` an array instead of string. If `g:fixmyjs_rc_filename` exists and it's a string, we could reassign it an array with the string.